admin/data/sql/migrations/20190221174823.do.add-age-reason-pupil-table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE [mtc_admin].[pupilAgeReason] (
+  id int IDENTITY (1,1) NOT NULL,
+  pupil_id int NOT NULL,
```

Comment:
Sorry, I missed this earlier.  Shouldn't `pupil_id` be UNIQUE?

Reply:
@jon-shipley yes and pushed.